examples/subnet.go

```diff
@@ -91,3 +93,94 @@ func DeploySubnet() {
 	blockchainID, _ := newSubnet.Commit(*deployChainTx, wallet, true)
 	fmt.Printf("blockchainID %s \n", blockchainID.String())
 }
+
+func DeploySubnetWithLedger() {
+	subnetParams := getDefaultSubnetEVMGenesis()
+	newSubnet, _ := subnet.New(&subnetParams)
+	network := avalanche.FujiNetwork()
+
+	// Create keychain with a specific Ledger address. More than 1 address can be used.
+	//
+	// Alternatively, keychain can also be created from Ledger without specifying any Ledger address
+	// by stating the amount of AVAX required to pay for transaction fees. Keychain SDK will
+	// then look through all indexes of all addresses in the Ledger until sufficient AVAX balance
+	// is reached. For example:
+	//
+	// fee := network.GenesisParams().CreateBlockchainTxFee + network.GenesisParams().CreateSubnetTxFee
+	// ledgerInfo := keychain.LedgerParams{
+	//	 RequiredFunds: fee,
+	// }
+	//
+	// To view Ledger addresses and their balances, you can use Avalanche CLI and use the command
+	// avalanche key list --ledger [0,1,2,3,4]
+	// The example command above will list the first five addresses in your Ledger
+	//
+	// To transfer funds between addresses in Ledger, refer to https://docs.avax.network/tooling/cli-transfer-funds/how-to-transfer-funds
+	ledgerInfo := keychain.LedgerParams{
+		LedgerAddresses: []string{"P-fujixxxxxxxxx"},
```

Comment:
can you make this example executable by first looking current ledger addresses for some indices and a connected ledger device?

Reply:
LGTM but I do believe a working example is nice to have